core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -30,19 +30,19 @@ contract Liquidatable is PricelessPositionManager {
 
     struct LiquidationData {
         /** Following variables set upon creation of liquidation */
-        uint expiry; // When Liquidation ends and becomes 'Expired'
         address sponsor; // Address of the liquidated position's sponsor
         address liquidator; // Address who created this liquidation
         Status state; // Liquidated (and expired or not), Pending a Dispute, or Dispute has resolved
+        uint liquidationTime; // Time when liquidation is initiated, needed to get price from Oracle
         /** Following variables determined by the position that is being liquidated */
         FixedPoint.Unsigned tokensOutstanding; // Synthetic Tokens required to be burned by liquidator to initiate dispute
         FixedPoint.Unsigned rawLockedCollateral; // Collateral locked by contract and released upon expiry or post-dispute
         // Amount of collateral being liquidated, which could be different from
         // lockedCollateral if there were pending withdrawals at the time of liquidation
         FixedPoint.Unsigned liquidatedCollateral;
-        /** Following variables set upon a dispute request */
+        /* Set upon initiation of a dispute */
```

Comment:
nit: this styling is inconsistent with the other section breaks within this struct. Should be:
```
/** Set upon initiation of a dispute */
```

Reply:
Done.